multiversx_sdk_cli/cli_delegation.py

```diff
@@ -209,7 +216,16 @@ def stake_nodes(args: Any):
     cli_shared.send_or_simulate(tx, args)
 
 
+def _check_if_either_bls_keys_or_validators_file_are_provided(args: Any):
+    bls_keys = args.bls_keys
+    validators_file = args.validators_file
+
+    if not bls_keys and not validators_file:
+        raise errors.BadUsage("No bls keys or validators file provided")
```

Comment:
Optionally, can be changed to _do this, do that_: "Either specify the ..., or the ...".

Reply:
Changed